src/main/java/de/dennisguse/opentracks/content/provider/CustomSQLiteOpenHelper.java

```diff
@@ -345,8 +345,8 @@ private void upgradeFrom29to30(SQLiteDatabase db) {
 
         // TrackPoints
         db.execSQL("ALTER TABLE trackpoints ADD COLUMN type TEXT CHECK(type IN (-2, -1, 0, 1))");
-        db.execSQL("UPDATE trackpoints SET type = -2, latitude = NULL, longitude = NULL WHERE latitude = 200.0");
-        db.execSQL("UPDATE trackpoints SET type = 1, latitude = NULL, longitude = NULL WHERE latitude = 100.0");
+        db.execSQL("UPDATE trackpoints SET type = -2, latitude = NULL, longitude = NULL WHERE latitude = 200");
```

Comment:
That's not gonna work either. It should be 200 * 1E6 :wink:

Reply:
Grummel ;)